package.json

```diff
@@ -1,21 +1,24 @@
 {
   "name": "nemo-saucelabs",
-  "version": "2.0.1",
-  "description": "updates Saucelabs jobs with test information and test results",
+  "version": "3.0.0",
+  "description": "updates Sauce Labs jobs with test information and test results",
   "main": "index.js",
   "author": {
     "name": "kushang gajjar",
     "email": "g.kushang@gmail.com"
   },
   "scripts": {
-    "test": "grunt"
+    "test": "echo \"TODO: write tests\""
```

Comment:
This module is missing Tests from beginning. We should add them (probably next PR?)